### PR TITLE
Added issue_template.md

### PR DESCRIPTION
**Pull request by looterz**
Github now has support for issue templates, so that people no longer have to copy and paste the template from somewhere else. You can test it out by creating an issue on my [fork](https://github.com/looterz/ACE3/issues)!

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+**ArmA Version:** `x.xx`
```
Comment (jonpas) [inline issue_template.md]: Arma

**Comment by jonpas** [comment]
Nice find!

**Comment by esteldunedain** [comment]
Excellent. Of course we'd have to delete the content for our internal task tickets, etc, but it'd really help with bug reports. :+1:

**Comment by looterz** [comment]
Another cool thing they added is a link to CONTRIBUTING.md when you submit an issue.

CONTRIBUTING.md alert

**Comment by jonpas** [comment]
We are using that one already. :)

**Comment by jonpas** [comment]
Can you uppercase `ISSUE_TEMPLATE.md` as it says in the news post https://github.com/blog/2111-issue-and-pull-request-templates ?

**Comment by looterz** [comment]
There we go, should be uppercase now. Git doesn't track case, so I had to remove and re-add the file.

**Comment by jonpas** [comment]
I don't see a reason to really.

**Comment by bux578** [comment]
> Think ACE3 will end up using the new .github/ folder?

yeah, like that. All nice and tidy.

**Comment by esteldunedain** [comment]
@looterz, could you move the file inside the `.github/` folder please?

**Comment by jonpas** [comment]
Should probably put `CONTRIBUTING.md` in there as well then.

**Comment by esteldunedain** [comment]
Perfect, thanks @looterz. @bux578, all yours.

**Comment by bux578** [comment]
Done! Thanks @looterz. I've added `PULL_REQUEST_TEMPLATE` aswell

**Pull request merged**